### DISCUSSION TITLE
Release Candidate One for Jamulus 3.10.0 is out now!

### DISCUSSION THREAD

**Discussion by pljones** [jamulussoftware/jamulus#3134] [Announcements]
Yes, yes, I know many people have been wondering what we've been up to.  Full details of the change are here:
https://github.com/jamulussoftware/jamulus/releases/tag/r3_10_0rc1
where you can also find the platform installers, if you want to **TEST** the pre-release.
(The sign macOS build will be along shortly.)

This pre-release is awaiting completion of a number of items.  Specifically, we could do with some help with translations:
* _Norwegian Bokmål_ (https://github.com/jamulussoftware/jamulus/issues/3132) and
* _Korean_ (https://github.com/jamulussoftware/jamulus/issues/3133)

So, if you want to volunteer, do get in touch - comment on the specific issues above, please.

**Comment by rdica** [comment]
Testing on private servers, including IPv6 connectivity. :+1:

**Comment by pljones** [comment]
Mine currently are:

*Public*
* Any Genre 2: jamulus.drealm.info - 3.10.0rc1 (recording active)
* jazz.jamulus.io:22324 / Genre Jazz - 3.10.0rc1
* rock.jamulus.io:22424 / Genre Rock - 3.10.0rc1
* classicalfolk.jamulus.io:22524 / Genre Classical/Folk - 3.10.0rc1
* choir.jamulus.io:22724 / Genre Choral/Barbershop - 3.10.0rc1

So please let me know if there are any problems on the above.

*Less public*
* wip.drealm.info:56850 - pljones/jamulus options/add-comments-about-validation
* jamulusdirectory2.drealm.info:22224 - jamulussoftware/jamulus main
* jamulusdirectory6.drealm.info:22624 - jamulussoftware/jamulus main
* jamulusdirectory8.drealm.info:22824 - jamulussoftware/jamulus main

(plus one other active on 3.9.1, plus two inactive ones, one on 3.9.1, one on 3.10.0rc1)
